plugin/pkg/admission/podresourceallocation/admission.go

```diff
@@ -149,7 +149,10 @@ func (p *Plugin) Validate(attributes admission.Attributes, o admission.ObjectInt
 	case admission.Create:
 		// newly created pods must have ResourcesAllocated field either empty or equal to Requests
 		for _, c := range pod.Spec.Containers {
-			if c.ResourcesAllocated == nil ||
+			if c.Resources.Requests == nil {
```

Comment:
Will this admit work for guaranteed pods, which have only resources.limit set?  Or this admission is after an admission that will change resources.request for guaranteed pods?